### PR TITLE
Pre M69 fixes

### PR DESCRIPTION
**Pull request by saghul**
These fixes remove some deprecations and lay the ground for M69, but can be applied to M67.

### CODE REVIEW

File: android/src/main/java/com/oney/WebRTCModule/GetUserMediaImpl.java [hunk 1]
```diff
@@ -270,7 +270,7 @@ private void getUserMedia(
             tracks_.pushMap(track_);
         }
 
-        String streamId = mediaStream.label();
+        String streamId = mediaStream.getId();
```
Comment (paweldomas) [inline android/src/main/java/com/oney/WebRTCModule/GetUserMediaImpl.java:273]: I don't see `getId()` here ? https://github.com/jitsi/webrtc/blob/M63/sdk/android/api/org/webrtc/MediaStream.java

Comment (paweldomas) [reply android/src/main/java/com/oney/WebRTCModule/GetUserMediaImpl.java:273]: are we M67 currently ?

File: ios/RCTWebRTC/RTCVideoViewManager.m [hunk 2]
```diff
@@ -87,8 +87,6 @@ @implementation RTCVideoView {
  * Tells this view that its window object changed.
  */
 - (void)didMoveToWindow {
-  [super didMoveToWindow];
```
Comment (paweldomas) [inline ios/RCTWebRTC/RTCVideoViewManager.m:91]: asking as I'm not an iOS expert yet: why is there no need to call super exactly ?

Comment (paweldomas) [reply ios/RCTWebRTC/RTCVideoViewManager.m:91]: is there some magic there that would do it automatically ? or is the `UIView` just an interface/empty class ?

Comment (saghul) [reply ios/RCTWebRTC/RTCVideoViewManager.m:91]: According to the documentation there is no need to call it.

**Review by paweldomas — APPROVED** [review]

File: android/src/main/java/com/oney/WebRTCModule/GetUserMediaImpl.java (hunk 1 above)
Comment (saghul) [reply android/src/main/java/com/oney/WebRTCModule/GetUserMediaImpl.java:273]: Yes, we are on M67.

**Pull request merged**